5-documentation/guides/agents/intermediate/send-tokens/sending_tokens.py

```diff
@@ -16,7 +15,6 @@ class TransactionInfo(Model):
 alice = Agent(name="alice", seed="alice secret phrase")
 bob = Agent(name="bob", seed="bob secret phrase")
 
-fund_agent_if_low(bob.wallet.address(), min_balance=AMOUNT)
```

Comment:
This is the only place where we should keep the funding method since it is not funding the agent for registration but the wallet for transactions.